toolkit/tools/grapher/grapher.go

```diff
@@ -42,7 +42,6 @@ var (
 	repoFiles                     = app.Flag("repo-file", "Full path to a repo file").Required().ExistingFiles()
 	usePreviewRepo                = app.Flag("use-preview-repo", "Pull packages from the upstream preview repo").Bool()
 	disableDefaultRepos           = app.Flag("disable-default-repos", "Disable pulling packages from PMC repos").Bool()
-	toolchainManifest             = app.Flag("toolchain-manifest", "Path to a list of RPMs which are created by the toolchain. Will mark RPMs from this list as prebuilt.").ExistingFile()
```

Comment:
Why are we removing this option.
Maybe someone's dev workflow breaks with this ?
I think removing it from the makefile should be sufficient.

Reply:
It was added by my previous commit. With multiple changes this variable is never used inside the code. So removing it as unnecessary.